.github/workflows/auto-updates.yaml

```diff
@@ -208,9 +208,7 @@ jobs:
         done
 
         # capture logs for the module changes
-        errout=$(mktemp)
-        deplog=$(modlog . HEAD dirty 2>$errout)
-        cat $errout
+        deplog=$(modlog . HEAD dirty || true)
```

Comment:
note: I made the `|| true` change so that `modlog` if it fails won't halt this action

I also realize I didn't need to capture the stderr